hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1532,6 +1532,13 @@ message TenantRevokeAdminResponse {
     optional bool success = 1;  // TODO: Remove this field
 }
 
+message OmDBAccessInfo {
+  optional string principal = 1;
+  optional bool isAdmin = 2;
+  optional bool isDelegatedAdmin = 3;
+  optional string tenantId = 4;
```

Comment:
Just curious. Is there anything in particular about the order of these fields?

If not let's keep the original order. And `principal -> userPrincipal`.

```suggestion
  optional string tenantId = 1;
  optional string userPrincipal = 2;
  optional bool isAdmin = 3;
  optional bool isDelegatedAdmin = 4;
```

Reply:
Sure I can do that.